sig-autoscaling/annual-report-2021.md

```diff
@@ -0,0 +1,69 @@
+[sig-governance.md](https://git.k8s.io/community/committee-steering/governance/sig-governance.md)?
+  * Is your README accurate? have a CONTRIBUTING.md file?
+    * Yes, [README.md](https://github.com/kubernetes/community/blob/master/sig-autoscaling/README.md) is accurate (other than one link)
+    * We do not currently have a [CONTRIBUTING.md] in the community repo
+
+  * All subprojects correctly mapped and listed in [sigs.yaml](https://git.k8s.io/community/sig-list.md)?
+    * [Yes](https://github.com/kubernetes/community/tree/master/sig-autoscaling#subprojects)]
+
+  * What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping
+  recordings up to date/trends in community members watching recordings?
+    * The main SIG meeting has ~8 people on average. Meeting notes [are recorded](https://docs.google.com/document/d/1RvhQAEIrVLHbyNnuaT99-6u9ZUMp7BfkPupT2LAZK7w/edit#heading=h.mh9yomoq1p6v), no video recordings made.
+
+* How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being
+retired? Are OWNERS.md files up to date in these areas?
+  * The SIG maintains direct ownership of all of the listed subprojects, with all discussed in one weekly meeting.
+  * Updates for cloud provider implementations for the cluster autoscaler are delegated to the owners of these, with work cutting across these communicated both through SIG meetings and asynchronously through Github Issues/PR mentions.
+
+* When was your last monthly community-wide update? (provide link to deck and/or recording)
+  * 2020/08/20 - [Recording here](https://youtu.be/oDL3Kp5-9eM?t=776)
+
+## Membership
+
+* Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+  * Yes - though some owners of cloud provider implementations for CA may no longer be active.
+  * Currently no proactive process for pruning inactive members from OWNERs files.
+
+* How do you measure membership? By mailing list members, OWNERs, or something else?
+  * Active participants in SIG meetings and pull requests.
+
+* How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+  * No official process for measuring reviewer and approver bandwidth, stuck or controversial PRs are usually discussed at weekly Office Hours.
+  * Reviewer bandwidth is occasionally a restricting factor as we have a small group of approvers, with a small number of experts in each of the SIGs three main areas of focus (Horizontal Pod Autoscaling, Cluster Autoscaler, Vertical Pod Autoscaler).
+
+* Is there a healthy onboarding and growth path for contributors in your SIG?
+What are some activities that the group does to encourage this? What programs are you participating in to grow contributors
+throughout the contributor ladder?
+  * We currently don't actively participate in any programs to grow contributors through the contributor ladder.
```

Comment:
Not a blocker by any means, but this is something we can potentially help you in ContribEx. We've had success with various mentoring cohorts and other means to get people onboarded into areas that need help 👍

Reply:
Thanks, since this PR was put together and off the back of discussions around the annual report process in the SIG Chairs meeting I've been having some discussions with @LappleApple around the enhancements liaison process she's been starting.

There's definitely room for expanding this with the help of ContribEx, particularly in seeking to grow the pool of code reviewers and approvers, but focused on making this first step successful for now.